python/ray/worker.py

```diff
@@ -448,7 +448,9 @@ def get_object(self, object_ids):
                 self.plasma_client.fetch(plain_object_ids[i:(
                     i + ray._config.worker_fetch_request_size())])
             else:
-                print("plasma_client.fetch has not been implemented yet")
+                for object_id in object_ids_to_fetch[i:(
+                    i + ray._config.worker_fetch_request_size())]:
+                    self.local_scheduler_client.fetch(object_id)
```

Comment:
I think it's pretty important to batch these calls

Reply:
Yeah, I'll do that before it gets merged, just wanted to see if the tests pass right now.